### PR TITLE
hip monitoring

### PR DESCRIPTION
**Pull request by mtosi**
this PR is needed for adding plots to the tracking DQM
in order to monitor the hip in different set of events (w/ or w/o OOT pu, w/ or w/o IN pu, etc)
in the ZeroBias PD
and to monitor few observables as function of the BX, lumi and PU
as requested/suggested by @VinInn @rovere 

I tested both running
runTheMatrix -l limited
and a modification of the 136.732 tests
the change was needed because the test --as it is currently setup-- 
does not run the DQM configuration which is actually run @tier0

everything seems to work as expected

the needed 3 new "keys" in the AlCaRecoTriggerBitsRcd payload
should be added by arun.kumar@cern.ch as discussed w/ @mmusich 
as soon as the PR is integrated in 80x

the corresponding trigger paths should be integrated in the next days
as described in [https://its.cern.ch/jira/browse/CMSHLT-981](https://its.cern.ch/jira/browse/CMSHLT-981)

I really hope (and somehow request) to have this PR integrated w/in few days
because we need this plots by next week in the GUI for the studies related to the hip
as requested by the HIP task force
@dmitrijus @vanbesien 

thanks much

### CODE REVIEW

**Comment by mmusich** [comment]
please test
@mtosi please clarify if the additional keys for the trigger selections are needed in re-reco GT (=`offline` tag) and MC as well.

**Comment by mtosi** [comment]
ciao

the monitoring is needed on PromptReco and rereco, of course
but 2 of the 4 selections will be available only from the next fill ..
  mia

File: DQM/TrackingMonitor/python/V0Monitor_cfi.py [hunk 1]
```diff
@@ -44,5 +44,10 @@
             xmin  = cms.double( -0.5),
             xmax  = cms.double( 59.5),
       ),
+      lsPSet = cms.PSet(
+            nbins = cms.int32 ( 2000 ),
+            xmin  = cms.double(    0.),
+            xmax  = cms.double( 2000.),
```
Comment (slava77) [inline DQM/TrackingMonitor/python/V0Monitor_cfi.py]: 2000 LS is just under 13 hours. Is it enough?

Comment (VinInn) [inline DQM/TrackingMonitor/python/V0Monitor_cfi.py]: I think it is the current standard, and we never seen real problems.

Comment (mtosi) [inline DQM/TrackingMonitor/python/V0Monitor_cfi.py]: the code is making use of
`->getTH1()->SetCanExtend(TH1::kAllAxes);`
as in 
https://github.com/cms-sw/cmssw/pull/15447/files/d6861782b900228f801c8785a6c284fa2c6cda6b#diff-9a8083d3aef749bdf613af451ba7f731R170

**Comment by mtosi** [comment]
backport in 80x is https://github.com/cms-sw/cmssw/pull/15449
hope everything is fine
thanks much

@dmitrijus @vanbesien

**Comment by dmitrijus** [comment]
+1

**Comment by davidlange6** [comment]
Hi @mtosi @mmusich - looks like we need that GT for integration testing.

and @mtosi- could you propagate all of the fixes from the 80x version to this one? thx

**Comment by mtosi** [comment]
ciao David
81x and 80x should be really the same a part the configuration code Matti
updated in 81x
  mia

**Comment by mmusich** [comment]
@davidlange6 I am on it

**Comment by mmusich** [comment]
see: https://github.com/cms-sw/cmssw/pull/15460

**Comment by davidlange6** [comment]
please test
thanks @mmusich

**Comment by VinInn** [comment]
Any reason why not integrated in 81X yet?

**Comment by davidlange6** [comment]
please test
yes, there was - hopefully solved now

**Comment by davidlange6** [comment]
+1

**Comment by smuzaffar** [comment]
@mtosi , this is failing for gcc493 IBs https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc493/CMSSW_8_1_X_2016-08-22-1100/DQM/TrackingMonitor
can you please provide a fix

**Comment by smuzaffar** [comment]
https://github.com/cms-sw/cmssw/pull/15559 should fix the build error

**Comment by mtosi** [comment]
thanks @smuzaffar !
and apologies for the late reply, but I did not read your message yesterday :(

**Comment by mmusich** [comment]
what about 80x version?

**Pull request merged**